doc/source/api-reference/qibo.rst

```diff
@@ -859,6 +861,11 @@ Entropy
     :members:
     :member-order: bysource
 
+.. note::
+    ``validate`` flag allows the user to choose if the function will check if input :math:`\\rho` is Hermitian or not.
+    Default option is ``validate=False``, i.e. the assumption of Hermiticity, because it is the faster implementation
+    and, more inportantly, the functions are intended to be used on Hermitian inputs. When ``validate=True`` and
```

Comment:
```suggestion
    and, more importantly, the functions are intended to be used on Hermitian inputs. When ``validate=True`` and
```
or I would remove "more importantly" completely to keep it shorter.

Reply:
Fixed